docs/partner_editable/additional_info.adoc

```diff
@@ -105,45 +109,48 @@ This Quick Start requires geo-location data to be uploaded to the _geodata_ fold
 |longitude |string
 |===
 
-If this data is missing, API request to get meter outage information will return error.
+If this data is missing, any API request to get meter-outage information will return an error.
 
 
 === Start the AWS Glue ETL and ML training pipeline
 
-By default, the pipeline is configured to be triggered each day at 09:00 AM and process the newly arrived data in the landing zone S3 bucket. However, to process the uploaded data without wait, trigger the pipeline manually. To trigger the ETL pipeline, go to the AWS Glue console and select the 'Workflow' menu entry. The AWS Glue workflow orchestrates the different steps of the ETL process. After the workflow run has successfully completed, Model training state machine is triggered to build the ML model and prepare the data for energy usage forecast and anomaly.
+By default, the pipeline is configured to be triggered each day at 09:00 and to process the newly arrived data in the landing-zone S3 bucket. To process the uploaded data without waiting, you trigger the ETL pipeline manually. When you do, the AWS Glue workflow orchestrates the steps of the ETL process. Then, the model training state machine is triggered to build the ML model and prepare the data for forecasting energy usage and reporting on anomalies.
 
-Follow the below steps to start the ETL pipeline.
+To trigger the ETL pipeline manually, follow these steps.
 
-. Go to the AWS Glue Console, and select 'Workflows'.
+. Go to the AWS Glue console, and choose *Workflows*.
 +
 :xrefstyle: short
 [#glue_console]
-.AWS Glue Console
+.AWS Glue console
 image::../images/4_start_etl_workflow.png[AWS Glue Console]
 
-. Select the workflow and choose 'Run' from the drop-down-menu.
+. Select the workflow. 
+. On the *Actions* menu, choose *Run*.
 +
 :xrefstyle: short
 [#start_etl_workflow]
 .Start ETL workflow
 image::../images/5_start_etl_workflow.png[Start ETL workflow]
 
-. The Workflow should be indicated as 'Running' in the 'History' tab.
+. In the *History* tab, make sure that the workflow's run status is *Running*.
 +
 :xrefstyle: short
 [#running_workflow]
 .Running workflow
 image::../images/6_start_etl_workflow.png[Running workflow]
 
-TIP: If the workflow doesn't start and jumps directly to 'Completed' go back to step 2.
+TIP: If the workflow doesn't start and jumps directly to a run status of *Completed*, go back to step 2.
+
+//TODO Shivansh, In the preceding tip, how do we set the step number to automatically track with editing changes in the steps, as we do with figure references? (Please let me know so that I can make these edits myself in the future.)
```

Comment:
I don't think there is a mechanism to link to that step within the bulletd list. Atleast I couldn't find out in the ascii docs guide, if there is one.

Reply:
There's an [open issue on this](https://github.com/asciidoctor/asciidoctor/issues/3253) in the asciidoctor repo. For now, this is a moot point since I no longer see this tip in this doc.